lib/msf/ui/formatter/option_validate_error.rb

```diff
@@ -12,16 +12,32 @@ class OptionValidateError
         def self.print_error(mod, error)
           raise ArgumentError, "invalid error type #{error.class}, expected ::Msf::OptionValidateError" unless error.is_a?(::Msf::OptionValidateError)
 
-          if error.reasons.empty?
-            mod.print_error("#{error.class} The following options failed to validate: #{error.options.join(', ')}")
-          else
-            mod.print_error("#{error.class} The following options failed to validate:")
-            error.options.sort.each do |option_name|
-              reasons = error.reasons[option_name]
-              if reasons
-                mod.print_error("Invalid option #{option_name}: #{reasons.join(', ')}")
+          (0...error.options.length).each do |i|
```

Comment:
Would using `each` here work? I don't think you need the index but if you do what about `each_with_index` 
```suggestion
          error.options.each do |option|
```

Reply:
Got this updated 👍